_about/participating.md

```diff
@@ -93,11 +93,11 @@ and to get information on participation and links to the mailing list
 archives.
 
 -   [ARIA Working
-    Group](https://www.w3.org/WAI/ARIA/ "link to ARIA WG home page")
+    Group](https://www.w3.org/WAI/ARIA/)
     develops a suite of Accessible Rich Internet Applications resources,
     and accessible APIs and mappings.
--   [[Accessible Platform Architectures (APA) Working
-    Group]{style="text-decoration: underline;"}](https://www.w3.org/WAI/APA/)
```

Comment:
Not sure what this does but the styling directives are appearing on the resulting page.

Reply:
Obviously this does nothing :-D I guess there were some inline-styles on the old website that was added and tried to survive the transition. Thanks for catching!